.github/components/dictionary.txt

```diff
@@ -182,6 +188,7 @@ multifactor
 multinucleated
 myeloid
 natively
+NCAM
```

Comment:
Can you use the full gene name if it includes a number, or does that not work?
```suggestion
NCAM1
```

Reply:
Nope, doesn't work. `NCAM` still gets flagged as typo.